talos/local_runner.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python3
+
+
+"""
+Simulated local run of Talos - Hail Query using a local runtime
+Currently applies only to SV data as a minimal test case
```

Comment:
Is this still true?

Reply:
This is definitely not true, but I'll remove this script before merging. It was just a test case for the AIP-in a bottle use case